resources/views/layouts/app.blade.php

```diff
@@ -71,10 +71,12 @@
             	</a>
 
 	            <div class="dropdown-menu bg-primary">
-                    <a class="dropdown-item text-white bg-primary" href="/profile">Profile</a>
+                    <a class="dropdown-item text-white bg-primary" href="/profile"><i class="fa fa-address-card-o" aria-hidden="true"></i>&nbsp;Profile</a>
+                    <a class="dropdown-item text-white bg-primary" href="/wishlist"><i class="fa fa-check-circle-o" aria-hidden="true"></i>&nbsp;Wishlist</a>
```

Comment:
good decision of making a separate menu item for wishlist, instead of accessing it through the profile page